webapp/packages/core-cli/configs/jest.config.js

```diff
@@ -36,4 +36,6 @@ module.exports = {
     // https://mswjs.io/docs/migrations/1.x-to-2.x#cannot-find-module-mswnode-jsdom
     customExportConditions: [''],
   },
+  // unset all global variables
+  globals: {},
```

Comment:
it's `{}` by default, to disable globals you need to add `injectGlobals: false`

Reply:
unfortunatelly it leads to 83 of 83 test to being fallen cause something is wrong with imported `expect`. Even with all cleared caches and other stuff. This moduled `jest` is really painful to use (impossible it just dont work like supposed to) and I hope we gonna move to the next framework during coming tech sprint

Reply:
for now IDE is not allowing to use globals. so I beleive we are good to go here